app/queues/reconciler/reconciler-queue.server.ts

```diff
@@ -0,0 +1,44 @@
+import logger from '~/lib/logger.server';
+import { reconcilerQueue } from './reconciler-worker.server';
+
+declare global {
+  var __reconciler_queue_init__: boolean;
+}
+
+reconcilerQueue.on('error', (err) => {
+  logger.error('Reconciler encountered an error', err);
+});
+
+// function to add jobs
+export const addReconcilerJob = async () => {
+  if (process.env.NODE_ENV !== 'production') {
+    return;
+  } else if (global.__reconciler_queue_init__) {
+    // Only do this setup once in dev
+    return;
+  }
+  global.__reconciler_queue_init__ = true;
+
+  logger.info('Starting DNS reconciler queue');
+
+  const jobName = `reconciler-scheduler`;
+
+  try {
+    // Remove all previously existing repeatable jobs
+    // This is important because multiple repeatable jobs can exist and they persist
+    // within redis (even with the same key)
+    const repeatableJobs = await reconcilerQueue.getRepeatableJobs();
+    Promise.all(repeatableJobs.map(({ key }) => reconcilerQueue.removeRepeatableByKey(key)));
```

Comment:
I would `await` this so you don't race on 33 when adding a new one.

Reply:
There can only ever be 1, but you are right, that's cleaner